### PR TITLE
Add a test comment

### PR DESCRIPTION
**Pull request by HarryCallahan308**
I added a test comment just to try out Git.

### CODE REVIEW

**Comment by PLally** [comment]
verified

**Comment by PLally** [comment]
verify

File: lua/ulx/modules/sh/cfc_cleanup.lua [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// This is a test.
```
Comment (PLally) [inline lua/ulx/modules/sh/cfc_cleanup.lua:1]:
```suggestion
-- This is a test.
```
:wink:

**Comment by plally** [comment]
test

**Pull request closed without merging**